### PR TITLE
Ghufran/week3

### PR DESCRIPTION
**Pull request by ghifo**

### CODE REVIEW

File: homework-submission/react-hyf/week3/src/homeworkWeek3.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+const UserName = ({ name, surname }) => {
+  return (
+    <h1>
+      {name} {surname}
+    </h1>
+  );
+};
+
+const UserInfo = ({ gender, region, phone, birthday, email, password }) => {
```
Comment (ArcoMul) [inline homework-submission/react-hyf/week3/src/homeworkWeek3.js]: Nice function components!

**Comment by ArcoMul** [comment]
Scratch the rendering on one page, please do move the PreviousHomework component to the same React app.

If you have time also render them on one page, but the homework isn't really clear on that so also fine if you leave it.

**Comment by fedefu** [comment]
thanks for the feedback @ArcoMul. Can you please update @ghifo so we close this PR? Thanks!

**Comment by ghifo** [comment]
> Scratch the rendering on one page, please do move the PreviousHomework component to the same React app.
> 
> If you have time also render them on one page, but the homework isn't really clear on that so also fine if you leave it.

I prefer leaving my homework this way if you please.
I made all the homework as components in the homework of week 4. 
Thanks for the feedback and sorry for my latency!

**Pull request closed without merging**